lib/kdeproject_component.rb

```diff
@@ -42,8 +44,13 @@ def plasma
 
     private
 
-    def to_names(projects)
-      projects.collect { |project| project.split('/')[-1] }
+    def to_names(project_list)
+      project_list.collect! { |project| project.split('/')[-1] }
```

Comment:
This edits the caller's list. You probably should avoid this by not using collect! and assign a new variable instead. Editing a caller's object will almost certainly cause confusion further down the line.